### PR TITLE
Added Styling

### PR DESCRIPTION
**Pull request by urvashi-code1255**

### CODE REVIEW

**Comment by Vishal-raj-1** [comment]
Hey @urvashi-code1255 always send some screenshots/ video clip in case of UI changes. It will be easy to reviewed.

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+.aboutPara{
+    margin-bottom: 1rem;
+    line-height: 1.8rem;
+    font-size: 1.1rem;
+    color: #716e6e;
+}
+
+/* Cards */
+.card{
```
Comment (Vishal-raj-1) [inline assets/css/style.css:30]: Don't directly change bootstrap class. Give some other class or id and then change styling.

File: index.html [hunk 2]
```diff
@@ -9,61 +9,137 @@
...
+    <!-- Navbar -->
+    <nav id="Navbar" class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container-fluid">
+            <a class="navbar-brand text-light" href="#">Awesome Javscript</a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
```
Comment (Vishal-raj-1) [inline index.html:26]: Once you copy the template from bootstrap, always change its id. Something like that 

```suggestion
            <div class="collapse navbar-collapse" id="navbar">
```

File: index.html [hunk 3]
```diff
@@ -9,61 +9,137 @@
...
+                <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                    <li class="nav-item">
+                        <a class="nav-link active text-light" aria-current="page" href="#">Home</a>
+                    </li>
+                    <li class="nav-item">
```
Comment (Vishal-raj-1) [inline index.html:31]: why did you give active class to every <a> tag ?

File: index.html [hunk 4]
```diff
@@ -9,61 +9,137 @@
...
+
+
+    <section id="project">
+        <div class="container my-3 text-center">
+            <h1> Awesome Javascript Projects</h1>
+        </div>
+        <div class="container border py-4">
+            <div class="row row-cols-3">
+                <div class="card m-3" style="width: 21rem;">
+                    <img class="card-img-top" src="Images/Project1.PNG" alt="Card image cap">
```
Comment (Vishal-raj-1) [inline index.html:79]: Change image name by project title. Like Project1.PNG doesn't define the image. But note-app.PNG define it is image of note app project. 

```suggestion
                    <img class="card-img-top" src="Images/Note-app.PNG" alt="Card image cap">
```

File: index.html [hunk 5]
```diff
@@ -9,61 +9,137 @@
...
                 </div>
-            </div>
 
-            <div class="card m-3" style="width: 18rem;">
-                <div class="card-body">
-                    <h5 class="card-title">Reviews</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
-                        card's content.</p>
-                    <a href="./Reviews/index.html" class="btn btn-primary">Visit</a>
+                <div class="card m-3" style="width: 21rem;">
```
Comment (Vishal-raj-1) [inline index.html:111]: inline css not required.

**Review by Vishal-raj-1 — CHANGES_REQUESTED** [review]
Great Work @urvashi-code1255 !! I have suggest you some changes. Please resolve this !!

File: index.html (hunk 5 above)
Comment (urvashi-code1255) [reply index.html:111]: This inline css was of cards you picked.

Comment (Vishal-raj-1) [reply index.html:111]: yep !! We can remove this now !!

**Comment by Vishal-raj-1** [comment]
@urvashi-code1255 always show preview!!

**Review by Vishal-raj-1 — APPROVED** [review]
Great !!

**Pull request merged**